addons/udes_stock_cron/models/stock_picking.py

```diff
@@ -62,6 +62,7 @@ def _find_unreservable_moves(self, batch_size=1000, return_fast=False):
                 ["product_id", "product_uom_qty", "state"], load="_classic_write"
             )
 
+            location = batched_pickings.picking_type_id.default_location_src_id
```

Comment:
I think on same place where it was previously 
locations not in the batched_pickings , but in self i believe as they are of the same picking type.
Batched is only splitting in chunks i believe

Reply:
Third time lucky? ;-)

Reply:
My bad , sorry :(